README.md

```diff
@@ -16,6 +16,8 @@ Role Variables
 
 `redis_port` port that the redis instance listens on.
 
+`node_hipache_version` version of node-hipache that is to be installed. Defaults to `latest`
```

Comment:
Is there any convention in Ansible that variables should be prefixed with the name of the playbook? if there are, then we're not following them already, but it might be worth doing as we add new things.